packages/fast-components-styles-msft/src/utilities/fonts.ts

```diff
@@ -6,10 +13,27 @@ export interface FontWeight {
     bold: number;
 }
 
-export const fontWeight: FontWeight = {
+export const defaultFontWeights: FontWeight = {
     light: 100,
     semilight: 200,
     normal: 400,
     semibold: 600,
     bold: 700,
 };
+
+/**
+ * @deprecated
+ */
+export const fontWeight: FontWeight = defaultFontWeights;
+
+function weight(index: keyof FontWeight): DesignSystemResolver<string> {
```

Comment:
nitty I know  - but its really more of a key than an index

Reply:
yeesh